packages/core/upload/server/constants.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const ACTIONS = {
+  read: 'plugin::upload.read',
+  readSettings: 'plugin::upload.settings.read',
+  create: 'plugin::upload.assets.create',
+  update: 'plugin::upload.assets.update',
+  download: 'plugin::upload.assets.download',
+  copyLink: 'plugin::upload.assets.copy-link',
```

Comment:
I do have the same problem in the frontend: could we name these `[action]Assets`? I think so far only assets existed and therefore it made sense, but now we'll have to think about both of them.

Reply:
Since `assets` are called files, I'd propose `[action]Files` instead :)

Reply:
I like that :) 
However it depends on what we decide for the permissions since it was said that there is no RBAC for the folders. We may rely on the file permission. Example: if you can create a file, then you can also create a folder.

Reply:
No RBAC *yet* :) I found it confusing in the frontend because the file|asset permissions don't apply to folders (e.g. `update`) and would love to make clear they only apply to files. However, you are right: maybe we are re-using the file|asset permissions for folders at some point, and then it would be confusing too. Let's keep it for now 👍🏼